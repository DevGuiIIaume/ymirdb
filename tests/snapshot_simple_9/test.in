list snapshots
drop 1
snapshot
drop 1
list snapshots
drop 1
bye
