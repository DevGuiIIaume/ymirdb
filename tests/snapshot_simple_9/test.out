> no snapshots

> no such snapshot

> saved as snapshot 1

> ok

> no snapshots

> no such snapshot

> bye
